packages/contracts-bedrock/test/kontrol/scripts/make-summary-deployment.sh

```diff
@@ -56,9 +56,12 @@ if [ "$KONTROL_FP_DEPLOYMENT" = true ]; then
   SCRIPT_SIG="runKontrolDeploymentFaultProofs()"
 fi
 
+# Sender just needs to be anything but the default sender (0x1804c8AB1F12E6bbf3894d4083f33e07309d1f38)
+# Otherwise state changes inside of Deploy.s.sol get stored in the state diff under the default script address (0x7FA9385bE102ac3EAc297483Dd6233D62b3e1496)
+# Conflicts with other stuff that happens inside of Kontrol and leads to errors that are hard to debug
```

Comment:
What exactly are the conflicts? Would love any more details just to better understand this

Reply:
I genuinely have no idea